behavioral-questions.md

```diff
@@ -4,16 +4,16 @@ Answer the following questions. Use the Nugget + STAR method for each answer.
 
 1. Why did you choose a career in software development?
 
-- Nugget:
-- Situation:
-- Task:
-- Action:
-- Result:
+- Nugget: The reason I chose a career in software development, it started as a challenge ...
```

Comment:
This question can be sometimes a little tricky to spin in a way that is appealing to an interviewer.  Some things to consider if asked this would be how to focus on the more positive aspects of development.  An interviewer most likely knows development is difficult, but talking about the aspects that really draw you to the challenge of coding could make this answer stand out.